domain/customers/service.go

```diff
@@ -1,18 +1,30 @@
 package customers
 
 import (
+	"net"
+
+	"github.com/nuts-foundation/go-did/did"
 	nutsApi "github.com/nuts-foundation/nuts-node/vdr/api/v1"
 	"github.com/nuts-foundation/nuts-registry-admin-demo/domain"
-	"net"
 )
 
 type Service struct {
 	VDRClient  nutsApi.HTTPClient
 	Repository Repository
 }
 
-func (s Service) ConnectCustomer(id, name, town string) (*domain.Customer, error) {
-	didDoc, err := s.VDRClient.Create(nutsApi.DIDCreateRequest{})
+func (s Service) ConnectCustomer(id, name, town string, serviceProviderID *did.DID) (*domain.Customer, error) {
+	selfControl := true
```

Comment:
shouldn't self-control be false (and capabilityInvocation as well)? Such that the SP manages the DID document?